src/main/java/frc/robot/subsystems/flywheel/FlywheelCommandsBuilder.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.subsystems.flywheel;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import frc.utils.utilcommands.InitExecuteCommand;
+
+public class FlywheelCommandsBuilder {
+
+	private final Flywheel flywheel;
+
+	public FlywheelCommandsBuilder(Flywheel flywheel) {
+		this.flywheel = flywheel;
+	}
+
+	//@formatter:off
+	public Command setPower(double power) {
+		return new FunctionalCommand(
+				() -> {},
+				() -> flywheel.setPower(power),
+				interrupted -> flywheel.stop(),
+				() -> false,
+				flywheel
+		).withName("Set power to " + power);
+	}
+
+	public Command init(Rotation2d targetVelocity) {
```

Comment:
setVelocity

Reply:
💀 don't resolve without changing or explaining why 😠

Reply:
i resolved it while on a plane so i guess it didnt send my comment, herman told me to call it init